test/test_requirements.py

```diff
@@ -136,7 +136,7 @@ def test_requirements():
             "cve_bin_tool.cli",
             "--input-file",
             SCAN_CSV,
-            "--merge",
+            "--triage-input-file",
```

Comment:
This did not work as the test apparently intended: because only one `--input-file` parameter is used, previously the `SCAN_CSV` was entirely ignored and the test only checked that all vulnerabilities that were present in `TRIAGE_JSON`were indeed mitigated.

Because of this, the test never noticed that we depend on `plotly.js` which has an open CVE and did not appear in the triage file.